gpii/node_modules/flowManager/test/BrowserChannelTests.js

```diff
@@ -30,25 +30,35 @@ gpii.tests.flowManager.browserChannel.chromeUserToken = "chrome2";
 gpii.tests.flowManager.browserChannel.firefoxUserToken = "firefox";
 
 gpii.tests.flowManager.browserChannel.payloads = {
-    expectedChromeSettings: {
-        "screenReaderTTSEnabled": false,
-        "highContrastEnabled": true,
-        "invertColours": false,
-        "magnifierEnabled": true,
-        "magnification": 2,
-        "fontSize": "medium",
-        "simplifier": false,
-        "highContrastTheme": "white-black"
+    "org.chrome.cloud4chrome": {
+        settings: {
+            "screenReaderTTSEnabled": false,
+            "highContrastEnabled": true,
+            "invertColours": false,
+            "magnifierEnabled": true,
+            "magnification": 2,
+            "fontSize": "medium",
+            "simplifier": false,
+            "highContrastTheme": "white-black"
+        },
+        options: {
+            "path": "org.chrome.cloud4chrome"
+        }
     },
-    expectedFirefoxSettings: {
-        "screenReaderTTSEnabled": false,
-        "highContrastEnabled": true,
-        "invertColours": false,
-        "magnifierEnabled": true,
-        "magnification": 2,
-        "fontSize": "medium",
-        "simplifier": false,
-        "highContrastTheme": "black-white"
+    "org.mozilla.cloud4firefox": {
+        settings: {
+            "screenReaderTTSEnabled": false,
+            "highContrastEnabled": true,
+            "invertColours": false,
+            "magnifierEnabled": true,
+            "magnification": 2,
+            "fontSize": "medium",
+            "simplifier": false,
+            "highContrastTheme": "black-white"
+        },
+        options: {
+            "path": "org.mozilla.cloud4firefox"
```

Comment:
It's not an example, is the way for the core architecture to know the path where the settings for a certain solution id should be placed (https://github.com/GPII/universal/blob/master/gpii/node_modules/settingsHandlers/src/WebSocketsComponent.js#L106)